packages/x-charts-vendor/package.json

```
@@ -12,7 +11,8 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/mui/mui-x"
+    "url": "https://github.com/mui/mui-x.git",
```

It seems that it should be:

```suggestion
    "url": "https://git+github.com/mui/mui-x.git",
```

per https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository. I have updated all the codebase (Base UI, Material UI, Pigment CSS, Toolpad, MUI X).